en/advanced_config/prearm_arm_disarm.md

```diff
@@ -4,13 +4,47 @@ Vehicles may have moving parts, some of which are potentially dangerous when pow
-  - In this state you can move ailerons, flaps etc, but motors/propellers are locked.
-- **Armed:** Vehicle is fully powered, including motors/propellers.
+- **Pre-armed:** Motors/propellers are locked but actuators for non-dangerous electronics are powered (e.g. ailerons, flaps etc.).
+- **Armed:** Vehicle is fully powered. Motors/propellers may be turning (dangerous!)
 
-A [safety switch](../getting_started/px4_basic_concepts.md#safety_switch) (if available) is used as a precondition for arming the vehicle, and may also be used to enable the pre-armed state. When permitted, arming is enabled using an arming sequence, switch or MAVLink command. 
+A [safety switch](../getting_started/px4_basic_concepts.md#safety_switch) (if available) is used as a precondition for arming the vehicle, and may also be used to enable the pre-armed state.
 
-PX4 allows you to configure how prearming, arming and disarming work using parameters (which can be edited in *QGroundControl* via the [parameter editor](../advanced_config/parameters.md)), as described in the following sections.
+When permitted, arming is enabled using an [arming gesture](#arm_disarm_gestures), [arming switch](#arm_disarm_switch) or MAVLink command. 
+
+PX4 allows you to configure how pre-arming, arming and disarming work using parameters (which can be edited in *QGroundControl* via the [parameter editor](../advanced_config/parameters.md)), as described in the following sections.
+
+> **Note** Arming/disarming parameters can be found in [Parameter Reference > Commander](../advanced_config/parameter_reference.md#commander) (search for `COM_ARM_` and `COM_DISARM_*`).
+
+## Arming Gesture {#arm_disarm_gestures}
+
+By default the vehicle is armed and disarmed by moving the RC throttle/yaw stick into a particular position and holding for 1 second:
+- **Arming:** stick to *bottom right*.
+- **Disarming:** stick to *bottom left*.
+
+The required hold time can be configured using [COM_RC_ARM_HYST](#COM_RC_ARM_HYST).
+
+Parameter | Description
+--- | ---
+<span id="COM_RC_ARM_HYST"></span>[COM_RC_ARM_HYST](../advanced_config/parameter_reference.md#COM_RC_ARM_HYST) | Time that RC stick must be held in arm/disarm position before arming/disarming occurs (default: 1 second).
+
+## Arming Button/Switch {#arm_disarm_switch}
+
+An *arming button* or "momentary switch" can be configured to trigger arm/disarm *instead* of [gesture-based arming](#arm_disarm_gestures).
+The button should be held down for ([nominally](#COM_RC_ARM_HYST)) one second to arm (when disarmed) or disarm (when armed).
+
+A two position-switch can also be used for arming/disarming, where the respective arm/disarm commands are sent on switch *transitions*.
+
+> **Tip** Use of a two-position switch is not recommended because it is easy to get the switch into an inconsistent state (i.e. if arming is rejected, the switch will say armed even though the vehicle is not armed).
```

Comment:
```suggestion
> **Tip** Use of a two-position switch is not recommended because it is easy to get the switch into an inconsistent state (i.e. if arming is rejected, the switch will stay armed even though the vehicle is not armed).
```
Maybe it's worth noting this feature is intended for more of a race-style completely manual flying use case? An arm switch is the industry standard for racing drones.

Reply:
Thanks for that - I was wondering why we supported this. I've just changed this to say "Two-position arming switches are primarily used in/recommended for racing drones.". If you're in that business then you know that is the option to use. Otherwise not.